fire-hose-management-frontend/src/components/Order/OrderForm.jsx

```diff
@@ -81,9 +80,7 @@ const OrderForm = () => {
   }, [devices, devices.length] )
 
   useEffect( () => {
-    if( Object.keys( department ).length !== 0 &&
-      forename.length !== 0 &&
-      surname.length !== 0 ) {
+    if( Object.keys( department ).length !== 0 && firstname.length !== 0 && lastname.length !== 0 ) {
```

Comment:
Codacy has a fix for the issue: [Replace `(·Object.keys(·department·).length·!==·0·&&·firstname.length·!==·0·&&·lastname.length·!==·0` with `·(⏎······Object.keys(department).length·!==·0·&&⏎······firstname.length·!==·0·&&⏎······lastname.length·!==·0⏎···`](https://app.codacy.com/gh/StitZle/fire-hose-complete/pullRequest?prid=9542260)
```suggestion
    if (
```